### PR TITLE
Added isPlaying() method. Updated Readme to reflect new method.

### PR DESCRIPTION
**Pull request by Tidal-Wave**
Added method that returns true if currently playing, false otherwise. Was unsure of your particular PR requirements or if I needed to bump any version numbers or anything. Let me know if any changes are needed.

### CODE REVIEW

File: src/wavesurfer.js [hunk 1]
```diff
@@ -146,6 +146,10 @@ var WaveSurfer = {
         this.backend.isPaused() ? this.play() : this.pause();
     },
 
+    isPlaying: function () {
+        return this.backend.isPaused() ? false : true;
```
Comment (katspaugh) [inline src/wavesurfer.js]: I think the idiomatic way is to use double negation, `!!`.

File: README.md [hunk 2]
```diff
@@ -92,6 +92,7 @@ All methods are intentionally public, but the most readily available are the fol
  * `pause()` – Stops playback.
  * `play([start[, end]])` – Starts playback from the current position.  Optional `start` and `end` measured in seconds can be used to set the range of audio to play.
  * `playPause()` – Plays if paused, pauses if playing.
+ * `isPlaying()` – Returns true if currently playing, false otherwise.
```
Comment (katspaugh) [inline README.md]: The methods are alphabetically sorted, could you please move it higher?

**Comment by Tidal-Wave** [comment]
Ah sure, I'm on the train atm but I'll make those changes.

**Comment by katspaugh** [comment]
Cool, thanks!

**Pull request merged**